pkg/metric_generator/generator.go

```diff
@@ -75,7 +84,13 @@ func (g *FamilyGenerator) generateHeader() string {
 	header.WriteString("# HELP ")
 	header.WriteString(g.Name)
 	header.WriteByte(' ')
-	header.WriteString(g.Help)
+	// Will remove if-else after all metrics are attached with right
```

Comment:
Can you add "TODO" key to the comment, so it won't be forgotten? (and folks will find it when grepping for TODO)

Reply:
+1.